src/app/public/modules/adapter-service/adapter.service.ts

```diff
@@ -0,0 +1,166 @@
+import {
+  ElementRef,
+  Injectable,
+  Renderer2,
+  RendererFactory2
+} from '@angular/core';
+
+import {
+  SkyMediaBreakpoints
+} from '../media-query';
+
+/* tslint:disable */
+const SKY_TABBABLE_SELECTOR = [
```

Comment:
Why do we need to disable linting here?

Reply:
Leftover from previous code. Removed it.